src/content/docs/zh-cn/develop/Debug/index.mdx

```diff
@@ -1,15 +1,148 @@
 ---
 title: 调试
-description: 调试工作流程的技巧和窍门
+sidebar:
+  label: 概述
+  order: 10
+i18nReady: true
 ---
 
-import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
-<CardGrid>
```

Comment:
In order to ensure consistency with the original document format, I have deleted it here.

Reply:
it's ok, that's how it should be done. Always base on the original